src/functions/tarsUpload/domain/upload/TARSPayloadConverter.ts

```diff
@@ -103,4 +105,33 @@ export class TARSPayloadConverter implements ITARSPayloadConverter {
       passCertificate: passCompletion.passCertificateNumber,
     };
   }
+
+  private populateAdi3Mark(
```

Comment:
Just thinking from a future proof perspective, as we have now understood the Standard Checks flow too, could we name this function ```populateMark``` as likely we can re-use same logic for SC as with ADI3?